sdk/translation/azure-ai-translation-text/samples/sample_text_translation_client.py

```diff
@@ -18,22 +18,24 @@
         1) AZURE_TEXT_TRANSLATION_ENDPOINT - the endpoint to your Text Translation resource.
         Note: the endpoint must be formatted to use the custom domain name for your resource:
         https:\\<NAME-OF-YOUR-RESOURCE>.cognitiveservices.azure.com\
-                    
-    The create_text_translation_client_with_credential call requires additional variables:
+
         2) AZURE_TEXT_TRANSLATION_APIKEY - the API key to your Text Translation resource.
         3) AZURE_TEXT_TRANSLATION_REGION - the Azure Region of your Text Translation resource.
+        4) AZURE_TEXT_TRANSLATION_RESOURCE_ID - the Azure Resource Id path
 """
 
 import os
 
+from static_access_token_credential import StaticAccessTokenCredential
```

Comment:
Where is this coming from?

Reply:
That is our custom implementation for the token service. Simple example how to create the Cognitive Services token.

Reply:
Where is StaticAccessTokenCredential defined? If we tell users they need to create the credential themselves, it definitely feels like we need an example of how to do that.

Reply:
https://github.com/Azure/azure-sdk-for-python/blob/main/sdk/translation/azure-ai-translation-text/tests/static_access_token_credential.py